index.html

```diff
@@ -21,5 +21,22 @@
 		          <img id="drop-btn" alt="drop menu icon" src="./images/dropMenuIcon.png">	   	
 		    </nav>  
     </div>
+	<div id="headline-section">
+	<aside>
+		<h2 class="title">
+			Hey there. I’m Alex.<span id="info" class="title">I’m a software developer</span>
+		</h2>
+		<p>
+		   I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+		</p>
+		<div>
+          <img alt="github icon" src="./images/github.png" />
+          <img alt="linkedin icon" src="./images/linkedin.png" />
+          <img alt="anglelist icon" src="./images/anglelist.png" />
+          <img alt="twitter icon" src="./images/twitter.png" />
+          <img alt="medium icon" src="./images/medium.png" />
+		</div>
```

Comment:
- [ ] This list of icons should use a `ul` tag to group them, and `li` per icon; each icon should have a link (`a` tag). This is one of the requirements so it would be nice to implement it.